ApollosProject/apollos-apps
# feat: Flag an Activity in a Community

## ✏️ Description

- Adds the ability to flag content in a community
- hides globally hidden content on load after reaching report limit



## 🔬 To Test
1. Go to community and click three dot menu in upper right to report
2. click report in dialog
3. content should become hidden

## 📸 Screenshots

![image](https://user-images.githubusercontent.com/7557597/214881793-b84e1b77-081a-4d43-ab31-b45cb1439b7f.png)
![image](https://user-images.githubusercontent.com/7557597/214881844-51c64775-4779-48a7-b278-25f1097e775d.png)


## ⚠️ To-do Before Merge



## 📝 Checklist
